Xamarin.Forms.Core/MenuItem.cs

```diff
@@ -46,9 +46,9 @@ public object CommandParameter
 
 		[Obsolete("Icon is obsolete as of 4.0.0. Please use IconImageSource instead.")]
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public FileImageSource Icon
+		public ImageSource Icon
```

Comment:
But don t we have a `IconImageSource`  ?

Reply:
It doesn't work for font icons if you use `IconImageSource`, because it uses `Icon` property and it's a FileImageSource.

Check the reproduction link:
https://github.com/anotherlab/FontImageSourceMissingInMenuItem

This XAML will not draw an icon

```
<MenuItem Text="Icon Missing">
    <MenuItem.IconImageSource>
        <FontImageSource
            FontFamily="{DynamicResource MaterialFontFamily}"
            Glyph="{x:Static icons:MaterialDesignIconFont.InformationOutline}"
            Color="Black"/>
    </MenuItem.IconImageSource>
</MenuItem>
```